src/test/java/no/unit/nva/bare/AuthorityConverterTest.java

```diff
@@ -23,44 +20,22 @@
             "createAuthorityRequestToBare.json";
     public static final String INVERTED_NAME = "Unit, DotNo";
     public static final String HTTPS_LOCALHOST_PERSON = "https://localhost/person/";
-    public static final String HTTPS_LOCALHOST_PERSON_WITHOUT_TRAILING_SLASH = "https://localhost/person";
-    public static final String SYSTEM_CONTROL_NUMBER = "1";
-    private Environment mockEnvironment;
+
 
 
     /**
      * Initialize tests.
      */
     @BeforeEach
     public void setUp() {
-        mockEnvironment = mock(Environment.class);
-        when(mockEnvironment.readEnv(AuthorityConverter.PERSON_AUTHORITY_BASE_ADDRESS_KEY))
-                .thenReturn(HTTPS_LOCALHOST_PERSON);
-    }
-
-
-    @Test
-    public void testTrailingSlashInPersonAuthorityBaseAddress() throws IOException {
-        Environment mockEnvironmentWithoutTrailingSlash = mock(Environment.class);
```

Comment:
Test testing custom logic for handling host URIs with or without the final slash. Used standard libary to remove the need 
for such logic (AuthorityConverter lines 97-104).